src/App.js

```diff
@@ -1,19 +1,74 @@
 import React from 'react';
 import './App.css';
-
 import users from './api/users';
+import todos from './api/todos';
+import { TodoList } from './components/TodoList';
+import { AddTodoForm } from './components/AddTodoForm';
+
+const names = users.map(({ name, id }) => (
+  {
+    name, id,
+  }
+));
+const initialNextId = todos.reduce(
+  (current, next) => (current.id > next.id ? current : next),
+).id + 1;
+
+class App extends React.Component {
+  state = {
+    todoList: todos,
+    defaultSelectValue: '0',
+    nextTodoId: initialNextId,
+  }
+
+  createId = (id) => {
+    let nextId = id;
+
+    return () => {
+      const previousId = nextId;
+
+      nextId += 1;
+      this.setState({ nextTodoId: nextId });
+
+      return previousId;
+    };
+  }
+
+  handleTodoData = (title, userId) => {
+    const idGenerator = this.createId(this.state.nextTodoId);
+    const id = idGenerator();
+    const newTodo = {
+      userId: Number(userId), title, id, completed: false,
+    };
+
+    this.setState(prevState => (
+      { todoList: [...prevState.todoList, newTodo] }
+    ));
```

Comment:
пиши код таким образом чтобы его можно было поддерживать
если я захочу сделать что-то помимо todoList, мне нужно будет переписывать
```suggestion
    this.setState(prevState => ({
      todoList: [...prevState.todoList, newTodo]
    }));
```